app/views/welcome/index.html.erb

```diff
@@ -29,13 +45,58 @@
           </label>
           <select class="form-control" name="which_test" id="AB_select">
             <option value="search_match_length">Search Match Length</option>
+            <option value="format_boosting">Format Boosting</option>
           </select>
         </div>
 
+        <div class="form-group">
+          <label class="form-label radio-label" for="info_basic">
+            <%= radio_button_tag("info", "basic", true) %>
+            <span class="radio-tag">Basic Information</span>
+            <div class="control_indicator" ></div>
+          </label>
+        </div>
+
+        <div class="form-group">
+          <label class="form-label radio-label" for="info_enhanced">
+            <%= radio_button_tag("info", "enhanced") %>
+            <span class="radio-tag">Enhanced Information</span>
+            <div class="control_indicator" ></div>
+          </label>
+        </div>
+
+        <div class="enhanced-checkbox">
+          <% enhanced_options = [ "historical", "popularity", "content_id", "document_collections",
+                                  "mainstream_browse_pages", "organisations", "taxons", "people",
+                                  "policies", "specialist_sectors"] %>
+          <% enhanced_options.each do |option| %>
+            <div class="form-group enhanced-option-wrapper">
+              <%= check_box_tag(option, "on", false, class: "enhanced-option") %>
+              <label class="enhanced-labels" for="<%= option %>" >
+                <%= option.split("_").map{|x| x.capitalize}.join(" ") %>
+              </label>
+            </div>
+          <% end %>
+        </div>
+        
+        <noscript>
+          <div class="noscript-enhanced-checkbox">
```

Comment:
this looks like a duplication for the above.

Also, is it needed, when/who do we expect to use this page without JS?